creator/ingest_runs/tasks/validation_run.py

```diff
@@ -1,79 +1,163 @@
 import os
+import json
+import jsonpickle
 import logging
+from pprint import pformat, pprint
+
+import pandas
 import django_rq
 from django.conf import settings
 from django.core.files.base import ContentFile
 from django_s3_storage.storage import S3Storage
 
+from kf_lib_data_ingest.etl.extract.utils import Extractor
+from kf_lib_data_ingest.validation.data_validator import (
+    Validator as DataValidator,
+)
+from kf_lib_data_ingest.validation.reporting.markdown import (
+    MarkdownReportBuilder,
+)
+
 from creator.decorators import task
 from creator.ingest_runs.models import ValidationRun, ValidationResultset
+from creator.files.models import extract_config_path, Version
+from creator.analyses.file_types import FILE_TYPES
+from creator.analyses.analyzer import extract_data
 
+ROOT_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+EXTRACT_CFG_DIR = os.path.join(ROOT_DIR, "extract_configs", "templates")
 S3_STORAGE = "django_s3_storage.storage.S3Storage"
 
 logger = logging.getLogger(__name__)
 
 
-def validate_file_versions(validation_run):
+def clean_and_map(version):
     """
-    TODO - Replace later. Currently counts to 10 then returns
-
-    Upload validation results and report files to storage backend
+    Load tabular data from file version into a DataFrame and then use
+    ingest lib's extraction utility to clean the values and map the column
+    names to the standard set of concepts expected by the validator
     """
-    logger.info(f"Begin validation for {validation_run.versions.all()}")
-    results = {}
+    logger.info(f"Performing extraction to clean and map file {version}")
+
+    # Load ingest extract operations for this file
+    if not version.root_file:
+        raise Exception(
+            f"Validation failed. Version {version} missing root_file"
+        )
+    ec = extract_config_path(version)
+    if not ec:
+        logger.warning(
+            "⚠️  Could not find an extract config for file version {version} "
+            "with file type: {version.root_file.file_type}. This file will "
+            "not be included in validation."
+        )
+        return None
+    # Extract file content into a DataFrame
+    try:
+        df = pandas.DataFrame(extract_data(version))
+    except Exception as e:
+        logger.error(
```

Comment:
```suggestion
        logger.exception(
```

Reply:
@gsantia If I use logger.exception, the my message and the exception traceback will be logged. The task decorator already automatically logs the exception traceback so that's why I used logger.error here and re-raised the exception.